conda-forge.yml

```diff
@@ -1 +1,6 @@
 conda_forge_output_validation: true
+provider:
+  linux_ppc64le: default
+  linux_aarch64: default
```

Comment:
Note: this is the key to tell conda smithy at rerender time that "please consider other non-default platforms".